### PR TITLE
Success-message: new component

### PR DESCRIPTION
**Pull request by msendlakowski**
## Description
Added the `success-message` component and tests.

## Context
Primary concern is the nested button.  I'm not sure if there is a better way to nest this and still have all the button events available.

## References
Fixes: https://github.com/eBay/ebayui-core/issues/877

## Screenshots
Screen Shot 2019-09-20 at 1 26 25 PM
Screen Shot 2019-09-20 at 1 26 40 PM
Screen Shot 2019-09-20 at 1 26 48 PM

### CODE REVIEW

File: src/components/ebay-success-message/index.marko [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <span class="success-message__status" role="img" aria-label=data.iconA11yText></span>
+        <span>${data.title}</span>
+    </>
+
+    <p
+        class=["success-message__content", data.content.class]
+        style=data.content.style
+        if(data.content && data.content.renderBody)>
+        <invoke data.content.renderBody(out) />
+    </>
```
Comment (seangates) [inline src/components/ebay-success-message/index.marko]: I know this is an easy convention, but we usually close non-dynamic tags so there's no confusion.
```suggestion
    </p>
```

**Comment by ianmcburnie** [comment]
@msendlakowski take a look at the way @agliga is adding the CTA button for page notice module. The downside of what you currently have is that the developer has to know which class to add to the button.

**Comment by ianmcburnie** [comment]
This is on hold due to design.

**Comment by ianmcburnie** [comment]
@msendlakowski  I'll try and make sure to keep this branch updated, so that it doesn't get stale.

**Comment by ianmcburnie** [comment]
Going to close this one and reopen when we have further updates from design. I'll keep the branch updated as we go.

**Pull request closed without merging**